src/probeinterface/io.py

```diff
@@ -1700,6 +1700,15 @@ def read_openephys(
             raise Exception("NP_PROBE field not found in settings")
         return None
 
+    # In neuropixel plugin 0.7.0, the option for enabling/disabling probes was added.
+    # Make sure we only keep enabled probes.
+    if neuropix_pxi_version >= parse("0.7.0") and neuropix_pxi_version < parse("1.0.0dev0"):
+        np_probes = [probe for probe in editor.findall("NP_PROBE") if probe.attrib["isEnabled"] == "1"]
```

Comment:
Actually, this would still fail for Open Ephys GUI version > 0.9, since the `NP_PROBE` are not in editor, but in custom_parameters. See lines https://github.com/SpikeInterface/probeinterface/pull/308/files#diff-9b4f228b41ad6dd338c7783897b5b949ef3825523b703d0376e755318f935549R1689-R1696

So you could just use the already found `np_probes`
```suggestion
        np_probes = [probe for probe in np_probes if probe.attrib["isEnabled"] == "1"]
```